src/components/AreaPanel.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
+  import { selectedGeographyStore } from "../stores/stores";
+  $: selectedGeographyDisplayName = $selectedGeographyStore?.displayName;
+  $: selectedGeographyGeoType = $selectedGeographyStore?.geoType.toUpperCase();
+
+  $: open = false;
+  let q = "";
+  let inputRef;
+  const handleClick = () => {
+    // inputRef.focus();
+
+    open = !open;
+    // if (open) {
```

Comment:
discussed offline - this and other commented bits are stuff thats in the process of being changed/removed and will be finished in a later PR - aim here is to merge work already done to avoid jamming up dev